tools/rna_tools/vienna_rna/rnalfoldz.xml

```diff
@@ -0,0 +1,97 @@
+<tool id="rnalfoldz" name="RNALfoldz" version="2.1.6.0">
+    <description>calculates locally stable secondary structures of RNA</description>
+    <expand macro="requirements" />
+    <expand macro="version_command" />
+    <expand macro="stdio" />
+    <macros>
+        <token name="@EXECUTABLE@">RNALfold</token>
+        <import>macros.xml</import>
+    </macros>
+    <command>
+<![CDATA[
+        RNALfold -L $span -T $temperature -d$dangling -z$zthresh < $fasta_input | sed -r 's/(^[.()]*) +\((.*)\) +(.*) +z= +(.*)/\1\t\2\t\3\t\4/' > $out_file
+        #if $varExists('$advancedOptions.noconversion')
+            #if $advancedOptions.noconversion
+                --noconv
+            #end if
+            #if $advancedOptions.gquad
+                --gquad
+            #end if
+            #if $advancedOptions.nolp
+                --noLP
+            #end if
+            #if $advancedOptions.nogu
+                --noGU
+            #end if
+            #if $advancedOptions.noclosinggu
+                --noClosingGU
+            #end if
+            #if $advancedOptions.notetra
+                --noTetra
+            #end if
+        #end if
+]]>
+    </command>
+    <inputs>
+	<param format="fasta" name="fasta_input" type="data" label="FASTA file"/>
+        <param name="zthresh" size="6" type="float" value="0" label="threshold for z-score computation (-z)" help="-z"/>
+        <param name="span" type="integer" min="0" max="400" value="150" label="base pair span" help="Maximal distance between two paired bases. (-L)"/>
+        <param name="temperature" type="float" value="37.0" label="Temperature [°C]" help="-T"/>
+        <param name="dangling" type="select" label="how to treat dangling end energies" help="-d">
+            <option value="2" selected="true">unpaired bases participate in all dangling ends (2)</option>
+            <option value="0">ignore dangling ends (0)</option>
+            <option value="1">unpaired bases participate in one dangling end only (1)</option>
+            <option value="3">allow coaxial stacking (3)</option>
+        </param>
+        <conditional name="advancedOptions">
+            <param name="advancedSelector" type="select"  label="advanced options">
+                <option value="basic">basic Options</option>
+                <option value="advanced">advanced Options</option>
+            </param>
+            <when value="advanced">
```

Comment:
Bjoern indicated that double negatives can be confusing (https://github.com/bgruening/galaxytools/issues/269). I addressed some of them in the other PR you can steal it from there